index.js

```diff
@@ -7,7 +7,7 @@
 
 'use strict';
 
-const isNumber = require('is-number');
+const isNumber = (v) => (typeof v === "number" && v - v === 0) || (typeof v === "string" && Number.isFinite(+v) && v.trim() !== "");
```

Comment:
Any thoughts on using regex for this?
```
const isNumber = (v) => /^-?\d+$/.test(v);
```

It passes all other existing tests as well.

Reply:
I recommend testing it against is-number test suite as suggested before to make sure we're not breaking anything

Reply:
> Any thoughts on using regex for this?
> 
> ```
> const isNumber = (v) => /^-?\d+$/.test(v);
> ```
> 
> It passes all other existing tests as well.

[A regex would be way slower](https://perf.link/#eyJpZCI6IjB1azN4a2Fydml4IiwidGl0bGUiOiJGaW5kaW5nIG51bWJlcnMgaW4gYW4gYXJyYXkgb2YgMTAwMCIsImJlZm9yZSI6IiIsInRlc3RzIjpbeyJuYW1lIjoiVGVzdCBDYXNlIiwiY29kZSI6ImNvbnN0IGlzTnVtYmVyID0gKHYpID0%2BICh0eXBlb2YgdiA9PT0gXCJudW1iZXJcIiAmJiB2IC0gdiA9PT0gMCkgfHwgKHR5cGVvZiB2ID09PSBcInN0cmluZ1wiICYmIE51bWJlci5pc0Zpbml0ZSgrdikgJiYgdi50cmltKCkgIT09IFwiXCIpO1xuaXNOdW1iZXIoMTAwMDAwMCk7IiwicnVucyI6Wzg2NDUwMDAsMTA0NjIwMDAsMTE4NjAwMCwxNTk5NTAwMCw4NzMwMDAsMTgzNjAwMCwxMTIxMDAwMCwxODk4MDAwLDE4MDAwLDIyMDAwMCw1MTg2MDAwLDE2ODYwMDAwLDM3MjIwMDAsNzcxMDAwLDg3MDEwMDAsNjk2ODAwMCw1ODAwMDAsMzgwNDAwMCw1MTg4MDAwLDE3MzQwMDAsMTExNDAwMDAsNTgxNzAwMCw1NjYxMDAwLDI1NzAwMCw2MjAzMDAwLDI3NTAwMCwxMDIyMTAwMCwyNDk0MDAwLDQ3NTYwMDAsOTIxNzAwMCw3MjYwMDAsMTA0MDAwLDE1MDAwLDM5NzAwMDAsMTk5NzAwMCwxNjAwMDAwLDg0NzEwMDAsMjUyMjAwMCwzMDk3MDAwLDM0NjQwMDAsMTA2NzgwMDAsNDg1NTAwMCw1NzIxMDAwLDkwMDAsMzI0MzAwMCwxNzA0MDAwLDEwMDg3MDAwLDg1NzgwMDAsMTI4NjcwMDAsMjgzMDAwMCwxNzg5MDAwLDM2NTQwMDAsNTkzMDAwLDQ0MzIwMDAsMjY5MzAwMCw2NTY1MDAwLDI1NTgwMDAsMjMxNTAwMCwxMzU2MzAwMCw2MzAwMCw2MjI4MDAwLDE2ODAwMCwxMDY0MDAwLDY1ODEwMDAsOTQwMzAwMCwxNTEyMDAwLDM0NTAwMDAsOTY0NzAwMCwxNDk5MDAwLDExMjE1MDAwLDUzMjkwMDAsMjc2NjAwMCwxNTMwOTAwMCwxNzI0MzAwMCwxMjIxMTAwMCwxMjM2NDAwMCwxMDAwLDEwODcwMDAsNDIzMzAwMCwzNTIzMDAwLDI4NTYwMDAsMzM1NjAwMCwzNjI1MDAwLDIwOTQwMDAsMTY2NjAwMCwyNjg1MDAwLDMwNTYwMDAsNzg0MDAwLDE2NDYwMDAsNjgyMzAwMCwzMDM3MDAwLDE3NjQ5MDAwLDEwNzE0MDAwLDMwNDkwMDAsNDcxMjAwMCwxMzU5MDAwLDE0OTkwMDAsODI3NDAwMCwyMTY3NjAwMCwzMzAwMDAwXSwib3BzIjo1MTUzNTQwfSx7Im5hbWUiOiJGaW5kIGl0ZW0gODAwIiwiY29kZSI6ImNvbnN0IGlzTnVtYmVyID0gKHYpID0%2BIC9eLT9cXGQrJC8udGVzdCh2KTtcblxuaXNOdW1iZXIoMTAwMDAwMCk7IiwicnVucyI6WzU3NDEwMDAsMjExNzAwMCwxMDAwMCw3NjYxMDAwLDIxMjUwMDAsODI5MDAwLDUxMjcwMDAsNDAzMDAwLDUyMzUwMDAsMjY3MDAwLDEyNjcwMDAsMTAxNTYwMDAsMTUwODAwMCw5MDAwLDQ4NzMwMDAsMjc3MDAwMCw2MzgzMDAwLDE0OTkwMDAsNDM0NjAwMCwyNDcwMDAsNjM0NjAwMCwxNzQ3MDAwLDE5OTYwMDAsOTMzNTAwMCwzMzE4MDAwLDEwMDAsNjY2NjAwMCwxNDk5MDAwLDI1OTcwMDAsMTc0NzAwMCw2NTAwMCwxNzQ3MDAwLDEwMDAsMTc0NzAwMCwxNDkxMDAwLDMzMTAwMCwxNTU1MDAwLDEwMzkwMDAsMTQ4NTAwMCwxNjg4MDAwLDcyMTcwMDAsMTU1NTAwMCw1Mjk3MDAwLDEwMDAsMTEwNzAwMCwxMjcyMDAwLDYxNjkwMDAsMjc0MjAwMCw3MDQ4MDAwLDYyNDAwMCwyNjYwMDAsMTkwMDAsMzAyNzAwMCw3NzAwMDAsMTIwMzAwMCwxNzQ3MDAwLDgwMjAwMCw4MDUxMDAwLDM5MjEwMDAsMTAwMCwzNzEwMDAsNDg5NTAwMCwzMDcwMDAsMzAwMDAsNTE4MzAwMCwxNzQ3MDAwLDEzOTUwMDAsMTYwMTAwMCwxNzkwMDAsMTAwMCwxNTYwMDAsMTcxNTAwMCwxMDAwLDM3NzgwMDAsNTQ2OTAwMCw2MzA2MDAwLDMzMzMwMDAsMTY5NzAwMCwxMDAwLDkzMDAwMCw5NDUwMDAsOTQ3MDAwLDEwNzUwMDAsNTcxMDAwLDI5MzcwMDAsMTc0NzAwMCwxMTU2MDAwLDE3NDcwMDAsMTAzMjUwMDAsMjgwMDAsMTE0MzAwMCwxMDU5NDAwMCwyNzA4MDAwLDE1NTUwMDAsMTM5MDAwMCwxNjc4MDAwLDkwMDAsMTY5MzAwMCwxMTYxMDAwMCwxNDk5MDAwXSwib3BzIjoyNTYyNjUwfV0sInVwZGF0ZWQiOiIyMDI0LTA3LTMwVDE4OjMwOjI1LjcyNVoifQ%3D%3D)